test/format/trace.c

```diff
@@ -30,7 +30,10 @@
 
 #define TRACE_CONFIG_CMD(cmd, flag)        \
     if ((p = strstr(copy, cmd)) != NULL) { \
-        flag = true;                       \
+        if (strcmp(cmd, "all") == 0)       \
+            all = true;                    \
+        else                               \
+            FLD_SET(g.trace_flags, flag);  \
```

Comment:
I think we need to set field even for "all" case, `FLD_SET(g.trace_flags, flag);` shouldn't be in the else block. And if you do that you'll not need the `if (all) {` block down below at the line 71.